solution/infra/terraform.tfstate

```diff
@@ -0,0 +1,8 @@
+{
+  "version": 4,
+  "terraform_version": "0.15.0",
+  "serial": 260,
+  "lineage": "ffd5a989-dd8b-067f-1e8c-2234b902a237",
+  "outputs": {},
+  "resources": []
+}
```

Comment:
Do you think we should keep this file in the VCS?

Reply:
It was my bad, I should not keep the ".tfstate" file on VCS, I should have used Terraform backend to track this file.